unit_converter/__init__.py

```diff
@@ -0,0 +1,460 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import annotations
+
+import json
+import re
+import traceback
+from datetime import datetime
+from pathlib import Path
+from typing import Any
+from urllib.request import urlopen
+
+import albert
+import inflect
+import pint
+
+default_trigger = "convert "
+synopsis = "<amount> <from_unit> to <to_unit>"
+
+__doc__ = f"""
+Extension for converting units of length, mass, speed, temperature, time,
```

Comment:
it make no sense to list just _some_ units. rather generalize to pysical and financial units or just units or provide a link to a list of units. sorry to be nitpicking but trying to take the pov of a user.